Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -987,6 +987,21 @@ a fixed cookie value as a trade-off. This makes all applications on the host to
 select the same service endpoint for a given service with session affinity configured.
 To disable the feature, set ``config.sessionAffinity=false``.
 
+LoadBalancer Source Ranges Checks
+*********************************
+
+When a ``LoadBalancer`` service is configured with ``spec.loadBalancerSourceRanges``,
+Cilium's eBPF kube-proxy replacement restricts access from outside to the service
```

Comment:
nit, maybe better clarify: from outside the cluster (external world traffic)

Reply:
Yeah, added `from outside (e.g. external world traffic)`, as it can be a request from another VPC which does not qualify as "world".